operations/model.go

```diff
@@ -91,6 +91,23 @@ func NewClientSettings(fn string) (*ClientSettings, error) {
 	}
 	conf.LoadedFrom = path
 
+	// check for a local settings file
+	localPath, err := findConfigFilePath(".evergreen.yml")
```

Comment:
Won't `findConfigFilePath` find the user settings file if a local one is not present, causing us to unmarshal it twice?

Reply:
Yep, I've removed the call to this entirely.